ui/packages/catalog-ui-search/src/main/webapp/component/query-advanced/query-advanced.view.js

```diff
@@ -25,6 +25,26 @@ const store = require('../../js/store.js')
 const QuerySettingsView = require('../query-settings/query-settings.view.js')
 const user = require('../singletons/user-instance.js')
 
+import query from '../../react-component/utils/query'
+
+const fetchFacets = memoize(async attr => {
+  const json = await query({
+    count: 0,
+    cql: "anyText ILIKE ''",
```

Comment:
Just curious, how is this cql using with respect to facets?

Reply:
It only returns results that match the cql. i.e. Say you have three records with titles: ["Mexico City", "USA", "Mexico"]
The cql `"anyText ILIKE ''"` will return the array ["Mexico City", "USA", "Mexico"]
The cql  `"anyText ILIKE 'Mexico'"` will return the array ["Mexico City", "Mexico"]